tests/components/zwave/test_init.py

```diff
@@ -1062,20 +1062,19 @@ def test_print_config_parameter(self):
             assert mock_logger.info.mock_calls[0][1][3] == 2345
 
     def test_print_node(self):
-        """Test zwave print_config_parameter service."""
+        """Test zwave print_node_parameter service."""
         node1 = MockNode(node_id=14)
         node2 = MockNode(node_id=15)
         self.zwave_network.nodes = {14: node1, 15: node2}
 
-        with patch.object(zwave, 'pprint') as mock_pprint:
+        with patch.object(zwave, '_LOGGER') as mock_logger:
             self.hass.services.call('zwave', 'print_node', {
-                const.ATTR_NODE_ID: 15,
+                const.ATTR_NODE_ID: 14
             })
             self.hass.block_till_done()
 
-            assert mock_pprint.called
-            assert len(mock_pprint.mock_calls) == 1
-            assert mock_pprint.mock_calls[0][1][0]['node_id'] == 15
+            assert mock_logger.info.called
```

Comment:
This assert is quite dangerous: I think renaming called would still pass the test:
```python
assert mock_logger.info.something_dangerous
```